### PR TITLE
Adjust keepalive

### PR DESCRIPTION
**Pull request by kmakarovobjectstyle**
I've set keepAlive time to 1 second as we discussed earlier.
Also changed library id to 'lutung-kontur' and version to 0.0.9
And added kontur repository
Compiled version is already in the kontur repository

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,11 +3,11 @@
...
-    <version>0.0.9-SNAPSHOT</version>
+    <artifactId>lutung-kontur</artifactId>
+    <version>0.0.9</version>
     <name>lutung</name>
     <description>Mandrill API Client for Java</description>
-    <url>https://github.com/rschreijer/lutung</url>
+    <url>https://github.com/konturio/lutung</url>
     <inceptionYear>2013</inceptionYear>
     <organization>
         <name>Microtrip.it</name>
```
Comment (alexei-grigoriev) [inline pom.xml:13]: kontur?

File: src/main/java/com/microtripit/mandrillapp/lutung/model/MandrillRequestDispatcher.java [hunk 2]
```diff
@@ -61,7 +60,14 @@
 				.setSocketTimeout(SOCKET_TIMEOUT_MILLIS)
 				.setConnectTimeout(CONNECTION_TIMEOUT_MILLIS)
 				.setConnectionRequestTimeout(CONNECTION_TIMEOUT_MILLIS).build();
-		httpClient = HttpClients.custom().setUserAgent("/Lutung-0.1")
+		httpClient = HttpClients.custom()
+				.setKeepAliveStrategy(new ConnectionKeepAliveStrategy() {
+					@Override
+					public long getKeepAliveDuration(HttpResponse httpResponse, HttpContext httpContext) {
+						return 1;
```
Comment (alexei-grigoriev) [inline src/main/java/com/microtripit/mandrillapp/lutung/model/MandrillRequestDispatcher.java]: why not something longer? I suggest 1000

Comment (alexei-grigoriev) [reply src/main/java/com/microtripit/mandrillapp/lutung/model/MandrillRequestDispatcher.java]: or make it configurable

File: pom.xml [hunk 3]
```diff
@@ -41,10 +41,34 @@
...
-        <developerConnection>scm:git:git@github.com:rschreijer/lutung.git</developerConnection>
-        <url>http://github.com/rschreijer/lutung</url>
+        <connection>scm:git:git://github.com/konturio/lutung.git</connection>
+        <developerConnection>scm:git:git@github.com:konturio/lutung.git</developerConnection>
+        <url>https://github.com/konturio/lutung</url>
     </scm>
+    <repositories>
+        <repository>
+            <id>kontur</id>
+            <url>https://nexus.kontur.io/repository/common/</url>
```
Comment (alexei-grigoriev) [inline pom.xml]: Make sure this url is open for public

File: pom.xml [hunk 4]
```diff
@@ -41,10 +41,34 @@
...
+            </snapshots>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+        </repository>
+    </repositories>
+    <pluginRepositories>
+        <pluginRepository>
+            <id>kontur</id>
+            <url>https://nexus.kontur.io/repository/common/</url>
```
Comment (alexei-grigoriev) [inline pom.xml]: Make sure this url is open for public

**Review by alexei-grigoriev — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (kmakarovobjectstyle) [reply pom.xml:13]: Is the address of this fork not a https://github.com/konturio/lutung ?

File: src/main/java/com/microtripit/mandrillapp/lutung/model/MandrillRequestDispatcher.java (hunk 2 above)
Comment (kmakarovobjectstyle) [reply src/main/java/com/microtripit/mandrillapp/lutung/model/MandrillRequestDispatcher.java]: I think the lower is better to not give it a chance to persist. Think of the situation there it spams mandrill many times per second, it persists for another 1000ms and it never ends. To make it configurable we have to make it not a static initialization at least, it is a larger modification. Also adding a parameter to DAS.

File: pom.xml (hunk 3 above)
Comment (kmakarovobjectstyle) [reply pom.xml]: Agree. Will look for pdc version of it and replace. Please let me know if a pdc version (such a used in thumbnail lib) is not good too. And what will be better then.

File: src/main/java/com/microtripit/mandrillapp/lutung/model/MandrillRequestDispatcher.java (hunk 2 above)
Comment (skunitzerpdc) [reply src/main/java/com/microtripit/mandrillapp/lutung/model/MandrillRequestDispatcher.java]: I'm OK with 1000 and not configurable.

File: pom.xml (hunk 3 above)
Comment (skunitzerpdc) [reply pom.xml]: PDC Nexus endpoint for Kontur libraries is here:
http://devapps.pdc.org/nexus/content/repositories/kontur-snapshot/
http://devapps.pdc.org/nexus/content/repositories/kontur-release/

Credential has been send to Konstantin separately

File: pom.xml (hunk 4 above)
Comment (skunitzerpdc) [reply pom.xml]: PDC Nexus endpoint for Kontur libraries is here:
http://devapps.pdc.org/nexus/content/repositories/kontur-snapshot/
http://devapps.pdc.org/nexus/content/repositories/kontur-release/

Credential has been send to Konstantin separately

File: src/main/java/com/microtripit/mandrillapp/lutung/model/MandrillRequestDispatcher.java (hunk 2 above)
Comment (kmakarovobjectstyle) [reply src/main/java/com/microtripit/mandrillapp/lutung/model/MandrillRequestDispatcher.java]: fixed

File: pom.xml (hunk 3 above)
Comment (kmakarovobjectstyle) [reply pom.xml]: fixed

File: pom.xml (hunk 4 above)
Comment (kmakarovobjectstyle) [reply pom.xml]: fixed

**Pull request merged**